sopel/plugins/rules.py

```diff
@@ -56,6 +57,11 @@
     'URLCallback',
 ]
 
+if sys.version_info.major >= 3:
+    # TODO: remove when dropping Python 2.7
+    unicode = str
+
+
```

Comment:
Why is this ported over to here? Nothing references the name `unicode` that I can see.

Reply:
Oh weird. I think an initial version used that. I shall remove that!